cli/src/commands/logs.ts

```diff
@@ -0,0 +1,39 @@
+import { Command, flags } from '@oclif/command';
+
+export default class Logs extends Command {
+  public static description = 'Show logs';
+
+  public static examples = [
+    '$ shift-cli logs',
+    '$ shift-cli logs --follow',
+    '$ shift-cli logs --since 2018-03-09T22:12:21.861Z',
+    '$ shift-cli logs --since 3d',
+    '$ shift-cli logs --since 13hours',
+    '$ shift-cli logs --since 9s*',
```

Comment:
What is `s*`?

Reply:
Maybe a mistake?
I copy-pasted from here https://docs.google.com/document/d/1284H6IQl8mt4EGeW1UEVYqs4HZWgJfFWZ8aARA0gaJ4/edit

Reply:
Looks like a mistake

Reply:
fixed